.github/workflows/automatic-release.yml

```
@@ -7,6 +7,10 @@ on:
         default: 18
         required: false
         type: string
+    secrets:
+      USER_TOKEN:
+        description: Authentication for the semantic release action.
```

During the development of the action we decided to "hide" the semantic release naming under the hood of "automatic release"
```suggestion
        description: Authentication for the automatic release action.
```